CHANGES.rst

```diff
@@ -12,6 +12,11 @@ Changes to API
 - Add ``outlier_detection`` submodule with ``utils`` included
   from jwst. [#270] 
 
+ramp_fitting
+~~~~~~~~~~~~
+
+- Add the likelihood algorithm to ramp fitting.[#278]
+
```

Comment:
```suggestion
```
#284 switched change log handling to `towncrier`, so this change log entry should be a file in `changes/`:
```shell
echo "[ramp_fitting] Add the likelihood algorithm to ramp fitting." > changes/278.apichange.rst
```

(new PRs will have these instructions in the checklist)